### ISSUE TITLE
feat(theme): add black and white only

### ISSUE THREAD

**Issue by dariadia** [dariadia/daria-s-design-system#8]
### What
We can set up a black and white only content (as theme).

### Why
UX. WCAG.

We can set up theme detection and opt for the black and white if certain user device is detected. Otherwise, show the user default colours. Users can change the theme on their own via a button as well.

### How
- Add a new `theme` to bundle;
- Add colours for the theme.
